source/session.ts

```diff
@@ -1005,3 +1005,18 @@ export class Session {
     });
   }
 }
+
+export function prepared(
```

Comment:
Not a big fan of the name, but don't have a good suggestion from the top of my head. Since it's a literal, a common naming seems rather to be the "use" of the final string. Maybe `expression`? If we put it inside the session class (See separate comment) it would then become `session.expression` or `this.expression`.

Reply:
The term comes from [Prepared Statements](https://en.wikipedia.org/wiki/Prepared_statement) which is known from every SQL server.

It's what's normally used to avoid injection.

I can certainly do `expression` instead, but I think many developers know about prepared statements, if they are backend developers.

Let me know what your thoughts are.

Reply:
If we're adopting the naming from SQL, shouldn't we go with prepare? 

I'll summon @jimmycallin to see if he has any strong feelings one way or the other - if not I'm happy with either naming.

Reply:
Imo, we would like the naming to be equivalent to `gql`, `css`, or `sql`. All of these also provide prepared statements in one way or another, but have a name that describes what kind of query/language it is rather than what it does. I think it makes sense for us to follow this as well, which is why I prefer it to be named `expression` (in lack of a better naming for our query language).

Reply:
Alright, will go with expression 🙏